biorbd_optim/optimal_control_program.py

```diff
@@ -290,5 +296,40 @@ def solve(self):
         # Solve the problem
         return solver.call(arg)
 
-    def show(self):
-        pass
+    def _get_a_reduced_ocp(self):
+        reduced_ocp = copy(self)
+        del (
+            reduced_ocp.J,
+            reduced_ocp.V,
+            reduced_ocp.V_bounds,
+            reduced_ocp.V_init,
+            reduced_ocp.g,
+            reduced_ocp.g_bounds,
+            reduced_ocp.show_online_optim_callback,
+            reduced_ocp.symbolic_controls,
+            reduced_ocp.symbolic_states,
+        )
+        for nlp in reduced_ocp.nlp:
+            del (
+                nlp["model"],
+                nlp["x"],
+                nlp["u"],
+                nlp["X"],
+                nlp["U"],
+            )
+        return reduced_ocp
+
+    @staticmethod
+    def save(ocp, sol, name):
+        with open(name, "wb") as file:
+            pickle.dump({"ocp": OptimalControlProgram._get_a_reduced_ocp(ocp), "sol": sol}, file)
+
+    @staticmethod
+    def load(biorbd_model_path, name):
+        with open(name, "rb") as file:
+            data = pickle.load(file)
+            ocp = data["ocp"]
+            sol = data["sol"]
+            for nlp in ocp.nlp:
+                nlp["model"] = biorbd.Model(biorbd_model_path)
+        return (ocp, sol)
```

Comment:
Can you add a test that compares the results before and after loading?

Reply:
test ? in "load_eocar.py" or elsewhere ?

Reply:
in the "tests" folder

Reply:
Is this done? I would close this PR if it is